.travis.yml

```diff
@@ -31,6 +32,8 @@ matrix:
       env: "RAILS=4.1.8"
     - rvm: 2.1
       env: "RAILS=4.2.0"
+    - rvm: 2.1
+      env: "RAILS=4.2.1"
```

Comment:
@timoschilling 
There is no reason.
I thought it is a common test-flow for this gem )
Should I drop it?